requirements-devel.txt

```diff
@@ -27,3 +27,5 @@
 #
```

Comment:
@SamiHiltunen can you please remove lines 24-29 and add also repositories for invenio-collections and invenio-upgrader. Thanks